packages/core/src/parser/StyleParser.test.ts

```diff
@@ -200,11 +200,44 @@ where C := intersect ( A, B, Not(f) ) ;\
     const prog = `
     forall Set x; Set y where y := Baz(x) as foo {}
     `;
-    expect(parseStyle(prog).isErr()).toEqual(true);
+    const parsed = parseStyle(prog);
+    if (parsed.isErr()) {
+      expect(parsed.error.message)
+        .toEqual(`Error: Syntax error at line 2 col 43:
+
+      forall Set x; Set y where y := Baz(x) as
+                                            ^
+Unexpected as token: "as". Instead, I was expecting to see one of the following:
+
+    ws token
```

Comment:
Seems like this depends on how we name the tokens. `whitespace` and `newline` will certainly be more readable than `ws` and `nl`.

Reply:
Yep, I agree; thankfully this is a pretty easy fix!